java/vector/src/main/codegen/templates/DenseUnionVector.java

```diff
@@ -104,6 +105,7 @@
    * The index is the type id, and the value is the type field.
    */
   private Field[] typeFields = new Field[Byte.MAX_VALUE + 1];
+  private byte[] typeMapFields = new byte[Byte.MAX_VALUE + 1];
```

Comment:
Could you please write some comments about the indices & values of the array?

Reply:
done